### ISSUE TITLE
Better integration instructions

### ISSUE THREAD

**Issue by keithpickering** [MyBBStuff/MyAlerts#187]
Hi, my team is using MyAlerts and we're currently trying to get it working with OUGC Awards.

The instructions for integrating MyAlerts with MyBB plugins provide plenty of code samples, but there is limited information regarding where any of this code should go. For example, I don't know what the `global_start` hook is, how I can hook into it, or in which file I should do so. Can all of this code go inside the plugin's php file? None of this is really made as clear as it should be.

For a plugin that's so extensible, it would be nice if the extend process could be a little easier to understand.

**Comment by euantorano** [comment]
Hi,

The `global_start` hook is a Plugin Hook wishing MyBB itself - for more information, please see here: http://docs.mybb.com/1.8/development/plugins/#plugin-hooks

You can use hooks within a plugin file to modify core behaviour.

Sorry the documentation isn't much more clear, it's mostly intended for other plugin developers who are usually well versed in the way MyBB does things.

**Comment by keithpickering** [comment]
Thanks for the reply. I was able to get it working - I had to store the alert formatter in a separate php file with all the others. Initially I was trying to create the formatter in OUGC Awards' main plugin file, but this was giving me an error. I think adding this detail to the documentation would help make it a lot more clear.

Great job on the plugin by the way. Once you figure it out it becomes really useful. Thanks again!
